src/extensions/averagewaveforms/AverageWaveformsView/AverageWaveformView.tsx

```diff
@@ -31,6 +31,7 @@ const AverageWaveformView: FunctionComponent<Props> = ({ boxSize, plotData, args
     return (
```

Comment:
I'm a little hazy on the distinction between `AverageWaveformView.tsx` and `AverageWaveformView2.tsx`--I thought the former was deprecated, but I see that's not the case. What's the difference?

Reply:
It is deprecated. Accidentally modified it.